core/src/main/java/cz/xtf/core/openshift/PodPortForward.java

```diff
@@ -0,0 +1,55 @@
+package cz.xtf.core.openshift;
+
+import io.fabric8.kubernetes.api.model.Pod;
+import io.fabric8.kubernetes.client.LocalPortForward;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class PodPortForward implements AutoCloseable {
+
```

Comment:
Could we delete this empty line? Just to keep it consistent with other classes.

Reply:
Sure, we probably need a codestyle check then :)

Reply:
Created an issue https://github.com/xtf-cz/xtf/issues/104 for that.